ckanext/depositar_theme/templates/home/index/idx_topics.html

```diff
@@ -1,86 +1,87 @@
-      <div class="w-100"></div><!-- /.BREAK LG -->
-      <div class="col">
-        <a  href="group/people-and-self"><img class="img-circle mt-1" src="{{ h.url_for_static('/images/icon-09.svg') }}" alt="people-and-self"></a>
-        <h5 class="text-center">{{ _('People and self') }}</h5>
-      </div><!-- /.span1 topic -->
-      <div class="col">
-        <a  href="group/philosophy-and-thinking"><img class="img-circle mt-1" src="{{ h.url_for_static('/images/icon-10.svg') }}" alt="philosophy-and-thinking"></a>
-        <h5 class="text-center">{{ _('Philosophy and thinking') }}</h5>
-      </div><!-- /.span1 topic -->
-      <div class="w-100 d-sm-none"></div><!-- /.BREAK SM -->
-      <div class="col">
-        <a href="group/religion-and-belief-systems"><img class="img-circle mt-1" src="{{ h.url_for_static('/images/icon-11.svg') }}" alt="religion-and-belief-systems"></a>
-        <h5 class="text-center">{{ _('Religion and belief systems') }}</h5>
-      </div><!-- /.span1 topic -->
-      <div class="col">
-        <a href="group/society-and-social-sciences"><img class="img-circle mt-1" src="{{ h.url_for_static('/images/icon-12.svg') }}" alt="society-and-social-sciences"></a>
-        <h5 class="text-center">{{ _('Society and social sciences') }}</h5>
-      </div><!-- /.span1 topic -->
-      <div class="w-100 d-lg-none"></div><!-- /.BREAK LG -->
-      <div class="col">
-        <a  href="group/technology-and-applied-sciences"><img class="img-circle mt-1" src="{{ h.url_for_static('/images/icon-13.svg') }}" alt="technology-and-applied-sciences"></a>
-        <h5 class="text-center">{{ _('Technology and applied sciences') }}</h5>
-      </div><!-- /.span1 topic -->
-      <div class="col"></div><!-- /.span1 topic -->
-      <div class="w-100 d-sm-none"></div><!-- /.BREAK SM -->
-      <div class="col d-xs-none"></div><!-- /.span1 topic -->
-      <div class="col d-xs-none"></div><!-- /.span1 topic -->
-    </div>
+    <h1>{{ _('Topic') }}</h1>
+    <ul>
+      <li>
+        <a href="/group/general-reference">
+          <img src="/images/icon-01.svg" alt="general-reference" class="img-circle">
+          <h5>{{ _('General reference') }}</h5>
+        </a>
+      </li>
+      <li>
+        <a href="/group/culture-and-the-arts">
+          <img src="/images/icon-02.svg" alt="culture-and-the-arts" class="img-circle">
+          <h5>{{ _('Culture and the arts') }}</h5>
+        </a>
+      </li>
+      <li>
+        <a href="/group/geography-and-places">
+          <img src="/images/icon-03.svg" alt="geography-and-places" class="img-circle">
+          <h5>{{ _('Geography and places') }}</h5>
+        </a>
+      </li>
+      <li>
+        <a href="/group/health-and-fitness">
+          <img src="/images/icon-04.svg" alt="health-and-fitness" class="img-circle">
+          <h5>{{ _('Health and fitness') }}</h5>
+        </a>
+      </li>
+      <li>
+        <a href="{{ h.url_for('/group/history-and-events') }}">
```

Comment:
改為 Flask endpoint (route) name
其他比照調整

```suggestion
        <a href="{{ h.url_for('group.read', id='history-and-events') }}">
```